### DISCUSSION TITLE
Plane and target image detection to locate the 3D model

### DISCUSSION THREAD

**Discussion by hbaugmentedreality** [google/model-viewer#2612] [Q&A]
Hi Guys.   Thank you for this model-viewer, it is great!  
It is possible to add to model-viewer the option of, additionaly to plane detection, detect a target image, so the 3D model is positioned in a certain position in the plane?    It would be great.    So, you detect the plane and also an image to locate the 3D model in that coordinates over the plane.  
Let me know.   Thanks!!!

**Comment by elalish** [comment]
We've talked about this, but it's dependent on getting image tracking support into WebXR. Any update on that @klausw?

**Reply by hbaugmentedreality** [reply]
I would need to have some kind of reference point over the plane or surface to locate the 3D model...  Do you know if there is any way to do it? 
Thanks!!!

**Reply by elalish** [reply]
Coordinate offset will be simple enough, but what we're missing currently is image detection.

**Reply by hbaugmentedreality** [reply]
Coordinate offset, but from which point?   because the model is located in a random point over the plane/surface detected, right?
Thanks
